.github/actions/common/setup

```diff
@@ -8,9 +8,6 @@ git checkout -b $commit
 composer create-project --prefer-dist laravel/laravel test-app
 cd test-app
 
-# Scaffold Authentication
```

Comment:
Why did you remove this from GitHub actions? If you remove this there will be no auth scaffolding. This is what makes tests fail here: https://github.com/DCzajkowski/auth-tests/pull/28/checks?check_run_id=212056515